app_dart/test/src/datastore/fake_datastore.dart

```diff
@@ -173,7 +173,8 @@ class FakeQuery<T extends Model> implements Query<T> {
     for (FakeFilterSpec filter in filters) {
       final String filterString = filter.filterString;
       final Object value = filter.comparisonObject;
-      if (filterString.contains('branch =') || filterString.contains('pr =')) {
+      if (filterString.contains('branch =') ||
+          filterString.contains('head =')) {
```

Comment:
Why changing this line? Is it relevant to this PR?

Reply:
Yes. This is for test `update if statuses have changed since last update - multiple branches`. 

Fake_datastore doesn't deal with `filter` for queries by default. `head` is added here since we need to test results filtered by it. (somehow the old `pr` doesn't work out)